api/tests/Api/Periods/ReadPeriodTest.php

```diff
@@ -103,8 +103,8 @@ public function testGetSinglePeriodIsAllowedForManager() {
             '_links' => [
                 'camp' => ['href' => $this->getIriFor('camp1')],
                 'materialItems' => ['href' => '/material_items?period=%2Fperiods%2F'.$period->getId()], // TODO: fix encoding with https://github.com/ecamp/ecamp3/issues/2289
```

Comment:
Is this solved now? Can this TODO be removed?

Reply:
Yes, thanks. Removed the comments.